### DISCUSSION TITLE
MEM tune only stored stations

### DISCUSSION THREAD

**Discussion by AaronJowitt** [PE5PVB/TEF6686_ESP32#373] [Ideas]
Minor suggestion. When rotary tuning MEM presets, skip presets that have not had a station stored in them.  Allows for faster MEM tuning if not using many of the 99 preset positions.

**Comment by PE5PVB** [comment]
Can do, but you also need to be able to toggle the SKIP state, otherwise it will be a mess after a while. Let me think about it.

**Comment by AaronJowitt** [comment]
Sure. A 'delete/clear MEM' option may do for basic management?

**Comment by PE5PVB** [comment]
Done.
